container_images/validate-integtest/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM golang
+RUN go get -u github.com/jstemmer/go-junit-report
+
+FROM python:3.7.8-slim-buster
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:latest
+
+RUN pip3 install junit2html
+COPY --from=0 /go/bin/go-junit-report /go-junit-report
```

Comment:
is it easier to use python as the last 'from' to minimize these copies?

Reply:
We need several env (node, python, golang, sdk) from multiple image. So i think keep the SDK as the last should be fine.